ui/src/app/modules/teams/components/actions-header/actions-header.component.ts

```diff
@@ -21,6 +21,9 @@ import {ActionItemService} from '../../services/action.service';
 import * as moment from 'moment';
 import {Themes} from '../../../domain/Theme';
 
+const ASSIGNEE_PARSE_SYMBOL = '@';
+const ASSIGNEE_PARSE_REGEX = /(\@[a-zA-Z]+\b)/g;
```

Comment:
This regex only supports A-Z; what if someone wants to use, say, CDSIDs, which contain numbers?

Reply:
This has been fixed!